### PR TITLE
Textual swap from 23.11 to 24.05

### PR DESCRIPTION
**Pull request by gvnn3**

### CODE REVIEW

**Comment by jrtc27** [comment]
It's misleading to update the version number without actually updating the contents. At least when the version number is outdated people know to expect the contents to be outdated too.

File: src/benchmarking/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ You will need to reboot in order for this change to take effect.
 
 # Heap temporal memory safety
 
-As of CheriBSD 23.11, userlevel heap temporal safety is enabled by default with jemalloc as the memory allocator.
+As of CheriBSD 24.05, userlevel heap temporal safety is enabled by default with jemalloc as the memory allocator.
```
Comment (jrtc27) [inline src/benchmarking/README.md:64]: It would be less misleading to say "Since CheriBSD 23.11" for these

File: src/packages/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ They have different levels of completeness, maturity, security, and support.
 extends FreeBSD ports, a collection of over 34,000 third-party
 software adaptations to FreeBSD, with CHERI- and CheriBSD-specific patches.
 The
-[releng/23.11](https://github.com/CTSRD-CHERI/cheribsd-ports/tree/releng/23.11)
+[releng/24.05](https://github.com/CTSRD-CHERI/cheribsd-ports/tree/releng/23.11)
```
Comment (jrtc27) [inline src/packages/README.md:12]: Update the URL too

File: src/features/temporal.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Userlevel heap temporal memory safety (experimental)
 
-CheriBSD 23.11 incorporates support for userlevel heap temporal memory
+As of CheriBSD 23.11 there has been support for userlevel heap temporal memory
```
Comment (jrtc27) [inline src/features/temporal.md:3]: Why not make this Since to match all the others that you just changed?..

**Comment by jrtc27** [comment]
Installation section still needs updating as the installer has definitely changed some of its dialogs, and the benchmark ABI kernel needs documenting in the kernel ABIs section. Maybe other things are outdated too, those are just the most obvious ones that spring to mind. Someone needs to own doing this properly, not just doing the easy mostly-automated part.

**Comment by brooksdavis** [comment]
Rolled into #60

**Pull request closed without merging**